### ISSUE TITLE
Add i386 builds to android-3.18 job

### ISSUE THREAD

**Issue by lag-linaro** [kernelci/kernelci-core#288]
If you could also enable older GCCs (5, 6 and 7), that would also be great.

**Comment by lag-linaro** [comment]
Is anyone working on KernelCI these days?

This is still very much needed.

**Comment by gctucker** [comment]
Yes there are people working on KernelCI, but the priority for kernelci.org is the upstream kernel and branches that contribute to it rather than 3.18 Android kernels.

**Comment by lag-linaro** [comment]
What does that mean in real terms?  Will this ever be fixed?

Arguably, heavily maintained (android-3.18 updated every day) Stable kernels are just as important, if not more important as the Mainline kernel.  Since these contain the fixes for, and are more likely to be used in real products than the Mainline kernel releases.

**Comment by gctucker** [comment]
I think Lee was referring to his branch which acted more like a very-LTS 3.18 rather than a pure Android kernel. It was in his tree:
https://git.kernel.org/pub/scm/linux/kernel/git/lee/linux.git

I say "was" because I'm not sure whether it's still active.  Could you please check last time `lee_android_3.18` was built or the branch was updated?

**Comment by lag-linaro** [comment]
I hope I'm not speaking out of turn here, but I believe all Android 3.18 branches can be removed, since they are no longer in support and will not be updated at any time in the future.

**Comment by gctucker** [comment]
Thanks again for confirming, that's all clear now.
